Query                              Series       Model Count  GEO title                                                                                                                                                         GEO summary                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       GEO overall design                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    Current Gemma tag(s)                                                                                                                                                                                                                                                                          Comment(s)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             Strong case 
---------------------------------  ----------  ------------  ----------------------------------------------------------------------------------------------------------------------------------------------------------------  --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------  --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------  --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------  ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------  ------------
Juvenile idiopathic arthritis      GSE13501              21  Subtype-specific peripheral blood gene expression profiles in recent onset JIA                                                                                    Objective: A multi-center study of recent onset juvenile idiopathic arthritis (JIA) subjects prior to treatment with DMARDS or biologics was undertaken to identify peripheral blood gene expression differences between JIA subclasses and controls. Methods: PBMC from 59 healthy children and 136 JIA subjects (28 enthesitis-related arthritis[ERA], 42 persistent oligoarthritis, 45 RF- polyarthritis, and 21 systemic) were isolated on Ficoll. Poly-A RNA was labeled using NuGEN Ovation and gene expression profiles were obtained using Affymetrix HG-U133 plus 2.0 Arrays. Results: 9,501 differentially expressed probe sets were identified among JIA subtypes and controls (ANOVA, FDR 5%). Specifically, 193, 1036, 873 and 7595 probe sets were different between controls and ERA, persistent oligoarthritis, RF- polyarthritis and systemic JIA samples respectively. In persistent oligoarthritis, RF- polyarthritis and systemic JIA subtypes, up-regulation of gene associated with IL-10 signaling was prominent. A hemoglobin cluster was identified that was under-expressed in ERA patients but over-expressed in systemic JIA. The influence of JAK/STAT, ERK/MAPK, IL-2 and B cell receptor signaling pathways was evident in persistent oligoarthritis. In systemic JIA, up regulation of innate immune pathways, including IL-6, TLR/IL1R, and PPAR signaling were noted, along with down regulation of gene networks related to NK and T cells. Complement and coagulation pathways were up-regulated in systemic JIA with a subset of these components differentially-expressed in the other three subtypes. Conclusions: Expression analysis identified differentially expressed genes in PBMCs between subclasses of JIA early in disease and controls, thus providing evidence for immunobiologic differences between these forms of childhood arthritis.

Keywords: Gene expression. PBMC. Patients compared with controls.                                                                                                                                                                                                                                                                                                                                                                                                                     PBMC from 59 healthy children and 136 JIA subjects (28 enthesitis-related arthritis[ERA], 42 persistent oligoarthritis, 45 RF- polyarthritis, and 21 systemic) were isolated on Ficoll. Poly-A RNA was labeled using NuGEN Ovation and gene expression profiles were obtained using Affymetrix HG-U133 plus 2.0 Arrays                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                male; biotin; European; female; Asian; sexually immature stage; peripheral blood mononuclear cell; arthritic joint disease                                                                                                                                                                    In the Mondo Disease Ontology, "juvenile idiopathic arthritis" is a sub-class of "arthritic joint disease."                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Yes         
Juvenile idiopathic arthritis      GSE8650               21  Blood Leukocyte Microarrays to Diagnose Systemic Onset Juvenile Idiopathic Arthritis and Follow IL-1 blocade                                                      Systemic onset Juvenile Idiopathic Arthritis (SoJIA) represents up to 20% of Juvenile Idiopathic Arthritis (JIA). We have previously reported that this disease is Interleukin 1 (IL1)-mediated, and that IL-1 blockade results in clinical remission in the majority of patients. The diagnosis of SoJIA, however, still relies on clinical findings as no specific diagnostic tests are available, which leads to delays in the initiation of specific therapy. To identify specific diagnostic markers, we analyzed gene expression profiles in 19 pediatric patients with SoJIA during the systemic phase of the disease (fever and/or arthritis), 25 SoJIA patients with no systemic symptoms (arthritis only or no symptoms), 39 healthy controls, 94 pediatric patients with acute viral and bacterial infections (available under GSE6269), 38 pediatric patients with Systemic Lupus Erythematosus (SLE), and 6 patients with a second IL-1 mediated disease known as PAPA syndrome. Statistical group comparison and class prediction identified genes differentially expressed in SoJIA patients compared to healthy children. These genes, however, were also changed in patients with acute infections and SLE. By performing an analysis of significance across all diagnostic groups, we generated a list of 88 SoJIA-specific genes (p<0.01 in SoJIA and >0.5 in all other groups). A subset of 12/88 genes permitted us to accurately classify an independent test set of SoJIA patients with systemic disease. We were also able to identify a group of transcripts that changed significantly in patients undergoing IL-1 blockade. Thus, analysis of transcriptional signatures from SoJIA blood leukocytes can help distinguishing this disease from other febrile illnesses and assessing response to therapy. Availability of accurate diagnostic markers for SoJIA patients may allow prompt initiation of effective therapy and prevention of long-term disabilities.
Keywords: Disease state analysis                                                                                                                                                                                                                                                                                                                                                    123 RNA samples extracted from PBMCs were studied. For more details on the clinical information, please refer to the paper (PudMed ID...).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            female; European; biotin; male; Asian; Hispanic or Latin American; peripheral blood mononuclear cell; systemic juvenile idiopathic arthritis; pyogenic arthritis-pyoderma gangrenosum-acne syndrome; systemic lupus erythematosus                                                             In the Experimental Factor Ontology (EFO), "systemic juvenile idiopathic arthritis" is a sub-class of "juvenile idiopathic arthritis." This series is the only one in Gemma with this tag.                                                                                                                                                                                                                                                                                                                                                                                                                                                                             Yes         
Neuroblastoma                      GSE22785              15  Integrated genome, transcriptome and translatome profiling of neuroblastoma cells                                                                                 Microarray-based genome-wide measurements of copy number alterations and of transcriptome variations are proposed for neuroblastoma prognosis and possible treatment choice. Nonetheless, they lack to provide clues on a neglected layer of systems-level changes, the translatome, whose variations are defined by the activity of the translational regulatory machinery. Our study extends the conventional genome-wide approaches to translatome profiling in neuroblastoma, by means of polysomal sucrose gradient separation followed by microarray analysis. The panel of fourteen parental (not subcloned) neuroblastoma cell lines used in the study includes: CHP-134, SIMA, NB-69, LAN-1, KELLY, CHP-126, CHP-212, SK-N-BE(2), IMR-32, SK-N-AS, SK-N-SH, STA-NB-7, STA-NB-1, STA-NB-10 cells. Each cell line has been profiled with high resolution array CGH analysis for copy number changes, and for transcriptome and translatome variations. The integration of these three types of data sets obtained from the same cells can provide information on the impact in neuroblastoma of a defined pattern of genomic lesions on both transcriptional and translational alterations of gene expression.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             We analyzed the following human neuroblastoma cell lines: STA-NB-1, SK-N-AS, NB-69, KELLY, LAN-1, CHP-134, SK-N-SH, SK-N-BE(2), STA-NB-7, CHP-126, CHP-212, IMR-32, SIMA, STA-NB-10.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  LAN-1 cell; polyA RNA extract; total RNA; IMR-32 cell; SK-N-SH cell; RCB0487 cell; SIMA cell; KELLY cell; CHP-126 cell; SK-N-AS cell; RCB0480 cell; SK-N-BE(2) cell; neuroblastoma cell line; CHP-212 cell                                                                                    It would seem consistent to tag this study with neuroblastoma because other studies with this tag have profiled neuroblastoma cell lines.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              Yes         
Neuroblastoma                      GSE45587              15  Expression data for developing a GE-HTS signature for neuroblastoma differentiation                                                                               Gene expression data from BE(2)-C cells treated in triplicate with either vehicle (DMSO), 5 μM all-trans retinoic acid (ATRA), 1 mM valproic acid (VPA), or 5 μM ATRA + 1 mM VPA for 6, 24, or 72 hours. Genome-wide expression profiling was performed using Affymetrix U133A microarrays.
While cytotoxic chemotherapy remains the hallmark of cancer treatment, intensive regimens fall short in many malignancies, including high-risk neuroblastoma. One alternative strategy is to therapeutically promote tumor differentiation. We created a gene expression signature to measure neuroblast maturation, adapted it to a high-throughput platform, and screened a diversity oriented synthesis-generated small-molecule library for differentiation inducers. We identified BRD8430, containing a nine-membered lactam, an ortho-amino anilide functionality, and three chiral centers, as a selective Class I histone deacetylase (HDAC) inhibitor (HDAC1 > 2 > 3). Further investigation demonstrated that selective HDAC1/HDAC2 inhibition using compounds or RNA interference induced differentiation and decreased viability in neuroblastoma cell lines. Combined treatment with 13-cis retinoic acid augmented these effects and enhanced activation of retinoic acid signaling. Therefore, by applying a chemical genomic screening approach we identified selective HDAC1/HDAC2 inhibition as a strategy to induce neuroblastoma differentiation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 BE(2)-C cells were treated in triplicate with either vehicle (DMSO), 5 μM all-trans retinoic acid (ATRA), 1 mM valproic acid (VPA), or 5 μM ATRA + 1 mM VPA for 6, 24, or 72 hours. Genome-wide expression profiling was performed using Affymetrix U133A microarrays [HT-HG_U133A Early Access].                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     biotin; dimethyl sulfoxide; BE(2)-C cell; valproic acid; all-trans-retinoic acid                                                                                                                                                                                                              It would seem consistent to tag this study with neuroblastoma because other studies with this tag have profiled neuroblastoma cell lines.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              Yes         
Triple negative breast carcinoma   GSE19697              13  An expression profile that predicts the therapeutic response of the basal-like breast cancer to neoadjuvant chemotherapy                                          A gene expression signature characterizes expression data from breast cancer samples of patients with pathological complete response (pCR) or residual disease (RD) following the neoadjuvant trial.
Several gene expression profiles have been reported to predict breast cancer response to neoadjuvant chemotherapy. These studies often consider breast cancer as a homogeneous entity, although higher rates of pathologic complete response (pCR) are known to occur within the basal-like subclass. We postulated that profiles with higher predictive accuracy could be derived from a subset analysis of basal-like tumors in isolation. Using a previously described ‘‘intrinsic’’ signature to differentiate breast tumor subclasses, we identified 50 basal-like tumors from two independent clinical trials associated with gene expression profile data. 24 tumor data sets (included in this GEO submission) were derived from a 119-patient neoadjuvant trial at our institution and an additional 26 tumor data sets were identified from a published data set (Hess et al. J Clin Oncol 24:4236–4244, 2006). The combined 50 basal-like tumors were partitioned to form a 37 sample training set with 13 sequestered for validation. Clinical surveillance occurred for a mean of 26 months. We identified a 23-gene profile which predicted pCR in basal-like breast cancers with 92% predictive accuracy in the sequestered validation data set. Furthermore, distinct cluster of patients with high rates of cancer recurrence was observed based on cluster analysis with the 23-gene signature. Disease-free survival analysis of these three clusters revealed significantly reduced survival in the patients of this high recurrence cluster. We identified a 23- gene signature which predicts response of basal-like breast cancer to neoadjuvant chemotherapy as well as disease-free survival. This signature is independent of tissue collection method and chemotherapeutic regimen.

Keywords: Disease state analysis                                                                                                                                                                                                                                                                                                                                Core biopsies were obtained from 86 patients prior to neoadjuvant therapy out of which 70 fulfilled the requirements to undergo expression analysis (24 of these 70 were used in the published analysis). pCR was defined as no residual invasive disease in the breast or lymph nodes. Residual in situ carcinoma was also considered as pCR. RNA was extracted from snap frozen 14-gauge core samples obtained from pre-treatment tumors. Specimens containing more than 40% of tumor on histological examination were analyzed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    female; European; African American; biotin; anthracycline; zoledronic acid; chemotherapy                                                                                                                                                                                                      According to the Mondo Ontology entry for basal-like breast carcinoma, it is a "biologic subset of breast carcinoma defined by high expression of genes characteristic of basal epithelial cells, including KRT5 and KRT17, annexin 8, CX3CL1, and TRIM29, and usually by lack of expression of the estrogen receptor (ER), progesterone receptor (PR), and human epidermal growth factor receptor 2 (HER2)." There is not clear overlap, but it is not surprising that this study was scored highly.                                                                                                                                                                  No          
Triple negative breast carcinoma   GSE31519              13  A Clinically Relevant Gene Signature in Triple-Negative and Basal-Like Breast Cancer                                                                              Current prognostic gene expression profiles for breast cancer mainly reflect proliferation status and are most useful in ER-positive cancers. Triple-negative breast cancers (TNBCs) are clinically heterogeneous, and prognostic markers and biology-based therapies are needed to better treat this disease.

We assembled Affymetrix gene expression data for 579 TNBCs and performed unsupervised analysis to define metagenes that distinguish molecular subsets within TNBC. We used n=394 cases for discovery and n=185 cases for validation. Sixteen metagenes emerged that identified basal-like, apocrine and claudin-low molecular subtypes, or reflected various non-neoplastic cell populations including immune cells, blood, adipocytes, stroma, angiogenesis, and inflammation within the cancer. The expressions of these metagenes were correlated with survival and multivariate analysis was performed including routine clinical and pathological variables.

73% of TNBCs displayed basal-like molecular subtype that correlated with high histological grade and younger age. Survival of basal-like TNBC was not different from non-basal-like TNBC. High expression of immune cell metagenes was associated with good and high expression of inflammation and angiogenesis-related metagenes were associated with poor prognosis. A ratio of high B-cell and low IL-8 metagenes identified 32% of TNBC with good prognosis (HR 0.37, 95% CI 0.22-0.61; P<0.001) and was the only significant predictor in multivariate analysis including routine clincopathological variables.

We describe a ratio of high B-cell presence and low IL-8 activity as a powerful new prognostic marker for TNBC. Inhibition of the IL-8 pathway also represents an attractive novel therapeutic target for this disease.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       Analysis of primary breast cancer biopsies from patients before treatment. No replicates. No control or reference samples are included.

The set of 579 TNBCs includes: (1) 67 new GEO Samples (GSM782523-GSM782589), (2) 489 re-analyzed GEO Samples (see 'Relation' links below), and (3) 23 re-analyzed ArrayExpress Samples.

Cohorts:
HH = University of Hamburg
FRA = University of Frankfurt, adjuvant chemotherapy
FRA-2 = University of Frankfurt, neoadjuvant chemotherapy
FRA-3 = University of Frankfurt, no adjuvant chemotherapy

Data processing of the 579 TNBC Samples:

MAS5 values were taken from GEO if available. For samples with no MAS5 values, CEL files were downloaded from GEO and the affy package from Bioconductor was used to generate MAS5 values. Next, MAS5 values corresponding only to the 22283 probesets from the U133A array were compiled. Subsequently, normalization of MAS5 data was performed using the command line version of the program CLUSTER 3.0 (Michael Eisen; updated by Michiel de Hoon; http://bonsai.hgc.jp/~mdehoon/software/cluster/command.txt).

The following three steps were performed in the following order:
1. log2 transformation of MAS5 values
2. median centering of arrays
3. magnitude normalization of arrays

These three steps correspond to the following commands:
cluster.com filename -l
cluster.com filename -ca m
cluster.com filename -na

The resulting dataset, which is linked below as a supplementary file, was used for the subsequent analyses.   biotin; triple-negative breast carcinoma; breast cancer; breast; Sample Study                                                                                                                                                                                                                 It appears this tag was added to the dataset in Gemma after we identified datasets with this tag. There were 11 datasets with this tag when we downloaded data from Gemma. Now there are 16, including this one.                                                                                                                                                                                                                                                                                                                                                                                                                                                       Yes         
Neuroblastoma                      GSE137975             12  Human embryonic stem cell-derived neural crest model for neuroblastoma                                                                                            Neuroblastoma is a common childhood malignant tumor of neural crest origin with remarkable heterogeneity in outcomes. Amplification of the oncogene MYCN is strongly associated with highly malignant behaviour and poor prognosis. Here, we used human ESC-derived neural crest model to recapitulate the initiation of MYCN-driven neuroblastoma and to identify MYCN downstream effectors. Our results show that induced deregulation of MYCN in human neural crest progenitor cells is sufficient to induce tumors that recapitulate the pathological and molecular features of human MYCN-amplified neuroblastoma(MNA-NB). By using this platform, we are able to identify a group of 28 genes that are associated with MYCN expression level only in MNA-NB.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                RNA-sequencing of control NCSCs and NCSCs+MYCN                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Overexpression; Tumor-derived cell line; MYCN [human] v-myc myelocytomatosis viral related oncogene, neuroblastoma derived (avian); embryonic stem cell line cell; neural crest cell; neuroblastoma cell line                                                                                 It would seem consistent to tag this study with neuroblastoma because other studies with this tag have profiled neuroblastoma cell lines.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              Yes         
Neuroblastoma                      GSE8866               11  The undifferentiated phenotype in neuroblastoma depends on Cyclin D1 and CDK4 activity                                                                            Genomic aberrations of Cyclin D1 (CCND1) and CDK4 in neuroblastoma indicate that dysregulation of the G1 entry checkpoint is an important cell cycle aberration in this pediatric tumor. Here we report that analysis of Affymetrix expression data of primary neuroblastic tumors shows an extensive over-expression of Cyclin D1 and CDK4 which correlates with histological subgroups and prognosis respectively. Immunohistochemical analysis demonstrated an over-expression of Cyclin D1 in neuroblasts and a low Cyclin D1 expression in all cell types in ganglioneuroma. This suggests an involvement of G1 regulating genes in neuronal differentiation processes which we further evaluated using RNA interference against Cyclin D1 and its kinase partner CDK4 in several neuroblastoma cell lines. This resulted in pRb pathway inhibition as shown by an almost complete disappearance of CDK4 specific pRb phosphorylation; reduction of E2F transcriptional activity and a decrease of Cyclin A protein levels. The Cyclin D1 and CDK4 knock-down resulted in a significant reduction in cell proliferation, a G1 specific cell cycle arrest and moreover an extensive neuronal differentiation. Affymetrix microarray profiling of siRNA treated cells revealed a shift in expression profile towards a neuronal phenotype. Several new potential downstream players are identified. We conclude that neuroblastoma functionally depend on over-expression of G1 regulating genes to maintain their undifferentiated phenotype.
Keywords: Neuroblastoma, CCND1, Cyclin D1, CDK4                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  The Cell line IMR-32 at time point 0 and transiently transfected with siRNA against GFP, Cyclin D1 and CDK4 at time point 48 hours. All experiments are biological triplicates.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       biotin; IMR-32 cell; CCND1 [human] cyclin D1; CDK4 [human] cyclin-dependent kinase 4; Knockdown                                                                                                                                                                                               It would seem consistent to tag this study with neuroblastoma because other studies with this tag have profiled neuroblastoma cell lines.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              Yes         
Down syndrome                      GSE46562              10  RNA-seq in neurons derived from iPSCs in controls and patients with schizophrenia and 22q11 del                                                                   Individuals with 22q11.2 Deletion Syndrome (22q11.2 DS) are a specific high-risk group for developing schizophrenia (SZ), schizoaffective disorder (SAD) and autism spectrum disorders (ASD). Several genes in the deleted region have been implicated in the development of SZ, e.g., PRODH and DGCR8. However, the mechanistic connection between these genes and the neuropsychiatric phenotype remains unclear. To elucidate the molecular consequences of 22q11.2 deletion in early neural development, we carried out RNA-seq analysis to investigate gene expression in differentiating human neurons derived from induced pluripotent stem cells (iPSCs) of 22q11.2 DS SZ and SAD patients. Eight cases (ten iPSC-neuron samples in total including duplicate clones) and seven controls (nine in total including duplicate clones) were subject to RNA sequencing. Using a systems level analysis, differentially expressed genes/gene-modules and pathway of interests were identified. We observed ~2-fold reduction in expression of almost all genes in the 22q11.2 region in SZ (37 genes reached p-value < 0.05, 36 of which reached a false discovery rate < 0.05). Outside of the deleted region, 745 genes showed significant differences in expression between SZ and control neurons (p<0.05). Function enrichment and network analysis of the differentially expressed genes uncovered converging evidence on abnormal expression in key functional pathways, such as apoptosis, cell cycle and survival, and MAPK signaling in the SZ and SAD samples.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      Eight cases (ten iPSC-neuron samples in total including duplicate clones) and seven controls (nine in total including duplicate clones)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               induced pluripotent stem cell line cell; schizophrenia; neuron; schizoaffective disorder; velocardiofacial syndrome                                                                                                                                                                           This is not a Down syndrome dataset. The word "Syndrome" may have caught the model's attention. Additionally, iPSCs are often used to study Down syndrome as used in this study.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       No          
Juvenile idiopathic arthritis      GSE15573              10  Immunity and Defense Genes in Peripheral Blood Mononuclear Cells of Rheumatoid Arthritis patients                                                                 Large-scale gene expression profiling of peripheral blood mononuclear cells from Rheumatoid Arthritis (RA) patients could provide a molecular description that reflects the contribution of diverse cellular responses associated with this disease. The aim of our study is to identify peripheral blood gene expression profiles for RA patients, using Illumina technology, to gain insights into RA molecular mechanisms. The Illumina Human-6v2 Expression BeadChips were used for a complete genome-wide transcript profiling of peripheral blood mononuclear cells from 18 RA patients and 15 Controls. Differential analysis per gene was performed with one-way analysis of variance (ANOVA) and P values were adjusted to control the False Discovery Rate (FDR<5%). Genes differentially expressed at significant level between patients and controls were analyzed using Gene Ontology (GO) in the PANTHER database to identify biological processes. A differentially expression of 339 Reference Sequence genes (238 down-regulated and 101 up-regulated) between the two groups was observed. We identified a remarkably elevated expression of a spectrum of genes involved in Immunity and Defense in peripheral blood mononuclear cells of RA patients compared to Controls. This result is confirmed by GO analysis, suggesting that these genes could be activated systemically in RA. No significant down-regulated ontology groups were found. Microarrays data were validated by real time PCR in a set of nine genes showing a high degree of correlation. Our study highlighted several new genes that could contribute in the identification of innovative clinical biomarkers for diagnostic procedures and therapeutic intervention. Further studies on larger scale groups of patients should be performed with the same technology to replicate these results and to allow clinical stratification.                                                                                                                                                                                                                                                                                                                                                                                                                                                 33 samples analyzed corresponding to 18 Rheumatoid Arthritis Patients and 15 Controls                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 biotin; female; male; blood; peripheral blood mononuclear cell; rheumatoid arthritis                                                                                                                                                                                                          JIA and RA are fairly closely connected in the Experimental Factor Ontology.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           No          
Juvenile idiopathic arthritis      GSE36700              10  Gene expression profiles in synovial biopsies from patients with arthritis                                                                                        Rheumatoid arthritis (RA) is an inflammatory joint disorder that results in progressive joint damage when insufficiently treated. In order to prevent joint destruction and functional disability in RA, early diagnosis and initiation of appropriate treatment with Disease-Modifying Antirheumatic Drugs (DMARDs) is needed. However, in daily clinical practice, patients may initially display symptoms of arthritis that do not fulfil the classification criteria for a definite diagnosis of RA, or any other joint disease, a situation called “Undifferentiated Arthritis” (UA). Out of the patients with UA, 30 to 50% usually develop RA, and early identification of these remains a challenge.

At the present time, although several risk factors associated with the development of RA have been identified (6-9), a model that reliably predicts the probability of evolution of UA into RA in individual patients is lacking. In order to better identify early RA patients, an American College of Rheumatology (ACR)/European League Against Rheumatism (EULAR) collaboration recently developed new classification criteria. Although these criteria are more sensitive, the risk of over-diagnosis is an important issue to consider, especially in very early disease. In this context, the present study explores the feasibility of a molecular diagnosis of arthritis, based on the identification of disease-specific transcriptomic profiles in synovial biopsies from patients with arthritis according to the underlying condition.

In a previous study, we performed global analyses of gene expression in synovial biopsies from patients with RA, Systemic Lupus Erythematosus (SLE) and Osteoarthritis (OA), using high-density oligonucleotide spotted microarrays. We found that the gene expression profiles are strikingly different according to the underlying condition. Thus, the majority of the genes up-regulated in SLE are type I Interferon-inducible genes, as compared with the up-regulation genes involved in T cell and B cell activation in RA, and in extracellular matrix homeostasis in OA. Based on these results, similar analyses were performed in synovial biopsies from patients with seronegative arthritis (SA) and microcrystalline arthritis (MIC), in order to identify disease-specific molecular signatures.   GeneChip® Human Genome U133 Plus 2.0 Arrays (Affymetrix UK Ltd., High Wycombe, UK) were hybridized in monoplicates with 10 μg biotinylated cRNA, using synovial biopsy samples obtained in 4 patients with SA (based on the presence of psoriasis or x-ray evidence of sacro-iliacal joint involvement), 5 patients with MIC, based on the identification of urate or calcium pyrophosphate crystals in the synovial fluid, 7 patients with RA (ACR 1987 criteria), 4 patients with SLE (ACR criteria) and 5 patients with OA (based on x-ray evidence of osteoarthritis)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             male; biotin; female; synovial membrane of synovial joint  rheumatoid arthritis; arthritic joint disease; non-steroidal anti-inflammatory drug; osteoarthritis; systemic lupus erythematosus; colchicine                                                                                      JIA and RA are fairly closely connected in the Experimental Factor Ontology.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           No          
Bipolar disorder                   GSE54570               9  Expression data from human brain dorsolateral prefrontal cortex - including control samples and samples with major depression disorders (26 samples NY_BA9)       Major depressive disorder is a heterogeneous illness with a mostly uncharacterized pathology. Large scale gene expression (transcriptome) analysis and genome-wide association studies (GWAS) for single nucleotide polymorphisms have generated a considerable amount of gene- and disease-related information, but heterogeneity and various sources of noise have limited the discovery of disease mechanisms. As systematic dataset integration is becoming essential, we developed methods and performed meta-clustering of gene coexpression links in 11 transcriptome studies from postmortem brains of human subjects with major depressive disorder (MDD) and non-psychiatric control subjects. We next sought enrichment in the top 50 meta-analyzed coexpression modules for genes otherwise identified by GWAS for various sets of disorders. One coexpression module of 88 genes was consistently and significantly associated with GWAS for MDD, other neuropsychiatric disorders and brain functions, and for medical illnesses with elevated clinical risk of depression, but not for other diseases (See publication for details).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               26 total samples in 13 pairs were analyzed in postmortem tissue from the dorsolateral prefrontal cortex.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              biotin; Depression; dorsolateral prefrontal cortex; major depressive disorder                                                                                                                                                                                                                 ChatGPT: "Bipolar Disorder (BD) and Major Depressive Disorder (MDD) are both mood disorders that share some common features, particularly involving depressive episodes, but they are distinct conditions with different diagnostic criteria."                                                                                                                                                                                                                                                                                                                                                                                                                         No          
Neuroblastoma                      GSE18409               9  Expression data from Trk-overexpressing NB cell line SY5Y                                                                                                         Neurotrophin receptors play a pivotal role in the biology and prognosis of neuroblastoma (NB). While expression of TrkA/NTRK1 confers a good prognosis to NB patients, expression of TrkB/NTRK2 is associated with unfavorable outcome. We have transfected the neurotrophin-receptor null cell line SY5Y with either full-length TrkA or TrkB and performed transcriptional profiling to analyse the effects of Trk-expression without activation and in a time course after activation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         SY5Y transfected with either full length TrkA- or TrkB cDNA or vector control were cultivated in the absence or presence of their ligands NGF and BDNF, respectively. At the indicated time points cells were harvested and analysed using Affymetrix U95Av2 chips.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   biotin; SH-SY5Y cell; Overexpression; NGF [human] nerve growth factor (beta polypeptide); BDNF [human] brain-derived neurotrophic factor; NTRK2 [human] neurotrophic tyrosine kinase, receptor, type 2; NTRK1 [human] neurotrophic tyrosine kinase, receptor, type 1                          It would seem consistent to tag this study with neuroblastoma because other studies with this tag have profiled neuroblastoma cell lines.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              Yes         
Bipolar disorder                   GSE54567               8  Expression data from human brain dorsolateral prefrontal cortex - including control samples and samples with major depression disorders (28 samples BA9_M)        Major depressive disorder is a heterogeneous illness with a mostly uncharacterized pathology. Large scale gene expression (transcriptome) analysis and genome-wide association studies (GWAS) for single nucleotide polymorphisms have generated a considerable amount of gene- and disease-related information, but heterogeneity and various sources of noise have limited the discovery of disease mechanisms. As systematic dataset integration is becoming essential, we developed methods and performed meta-clustering of gene coexpression links in 11 transcriptome studies from postmortem brains of human subjects with major depressive disorder (MDD) and non-psychiatric control subjects. We next sought enrichment in the top 50 meta-analyzed coexpression modules for genes otherwise identified by GWAS for various sets of disorders. One coexpression module of 88 genes was consistently and significantly associated with GWAS for MDD, other neuropsychiatric disorders and brain functions, and for medical illnesses with elevated clinical risk of depression, but not for other diseases (See publication for details).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               28 total samples in 14 pairs were analyzed in postmortem tissue from the dorsolateral prefrontal cortex.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              biotin; Depression; dorsolateral prefrontal cortex; major depressive disorder                                                                                                                                                                                                                 ChatGPT: "Bipolar Disorder (BD) and Major Depressive Disorder (MDD) are both mood disorders that share some common features, particularly involving depressive episodes, but they are distinct conditions with different diagnostic criteria." The models likely identified similarities in the ways these conditions are portrayed in dataset descriptions.                                                                                                                                                                                                                                                                                                           No          
Bipolar disorder                   GSE54575               8  Expression data from human brain orbital ventral prefrontal cortex - including control samples and samples with major depression disorders (24 samples NY_BA47)   Major depressive disorder is a heterogeneous illness with a mostly uncharacterized pathology. Large scale gene expression (transcriptome) analysis and genome-wide association studies (GWAS) for single nucleotide polymorphisms have generated a considerable amount of gene- and disease-related information, but heterogeneity and various sources of noise have limited the discovery of disease mechanisms. As systematic dataset integration is becoming essential, we developed methods and performed meta-clustering of gene coexpression links in 11 transcriptome studies from postmortem brains of human subjects with major depressive disorder (MDD) and non-psychiatric control subjects. We next sought enrichment in the top 50 meta-analyzed coexpression modules for genes otherwise identified by GWAS for various sets of disorders. One coexpression module of 88 genes was consistently and significantly associated with GWAS for MDD, other neuropsychiatric disorders and brain functions, and for medical illnesses with elevated clinical risk of depression, but not for other diseases (See publication for details).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               24 total samples in 12 pairs were analyzed in postmortem tissue from the orbital ventral prefrontal cortex.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           biotin; prefrontal cortex; Depression; major depressive disorder                                                                                                                                                                                                                              ChatGPT: "Bipolar Disorder (BD) and Major Depressive Disorder (MDD) are both mood disorders that share some common features, particularly involving depressive episodes, but they are distinct conditions with different diagnostic criteria." The models likely identified similarities in the ways these conditions are portrayed in dataset descriptions. This is a sister study to GSE54567.                                                                                                                                                                                                                                                                       No          
Down syndrome                      GSE216875              8  Reduced LYNX1 and epilepsy phenotype-related changes in transcriptome of human iPSC-derived neural progenitors modeling fragile X syndrome                        Transcriptome analysis of RNA samples collected from human control and FXS iPS cell-derived neural progenitors at day 1 and day 7 of differentiation
Lack of fragile X mental retardation protein results in fragile X syndrome (FXS), which is the most common inherited intellectual disability syndrome and serves as an excellent model disease to study molecular mechanisms behind neuropsychiatric comorbidities. We compared the transcriptomes of human neural progenitors (NPCs) generated from patient-derived induced pluripotent stem cells (iPSCs) of three FXS and three control male donors. Altered expression of RAD51C, PPIL3, GUCY1A2, MYD88, TRAPPC4, LYNX1, and GTF2A1L in FXS NPCs suggested changes related to triplet repeat instability, RNA splicing, testes development, and pathways previously shown to be affected in FXS. LYNX1 is a cholinergic break of tissue plasminogen activator (tPA)-dependent plasticity, and its reduced expression was consistent with augmented tPA-dependent radial glial process growth in NPCs derived from FXS iPSC lines. An analysis of gene expression in LYNX1-related signaling pathways revealed that NPCs derived from an FXS male with concomitant epilepsy differed from the other FXS NPCs. The differently expressed genes comprised several epilepsy-related genes, including genes shown to cause the auditory epilepsy phenotype in the murine model of FXS. Functional enrichment analysis highlighted regulation of insulin-like growth factor pathway in human NPCs modeling FXS with epilepsy. Our results link early gene expression changes of FXS NPCs with the pathogenesis of FXS and comorbid epilepsy.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    In this study, we analyzed gene expression profile of human iPS cell-derived neural progenitors reprogrammed from somatic cells of 3 healthy subjects and 3 patients diagnosed with FXS using the Affymetrix ClariomTM D Human array.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 biotin; neural progenitor cell; fragile X syndrome                                                                                                                                                                                                                                            This is not a Down syndrome dataset. The word "syndrome" may have caught the model's attention. Additionally, iPSCs are often used to study Down syndrome as used in this study.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       No          
Parkinson's disease                GSE33298               8  Gene expression analysis of human iPSC generated from pathogenic LRRK2 (G2019S) mutation bearing patients                                                         Genetic mutations on leucine-rich repeat kinase 2 (LRRK2) have been associated with an increased risk of Parkinson's disease. The Gly2019Ser (G2019S) mutation on LRRK2 gene is a relatively common cause of familial Parkinson's disease in Caucasian population. In this study, we generated human induced pluripotent stem cell (iPSC) lines from LRRK2 (G2019S) bearing patient fibroblasts by cell reprogramming.
We performed global gene expression profiling of LRRK2 (G2019S) heterozygous and homozygous patient iPSC lines, and the corresponding fibroblast lines they originated from. An age-matched wildtype human fibroblast line and H1 human embryonic stem cell (ESC) line were used as controls.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Microarray gene expression profiling was done to: (1) Compare global gene expression differences between wildtype fibroblasts and fibroblasts from patients bearing homozygous and heterozygous LRRK2 (G2019S) mutation; (2) Compare global gene expression differences between wildtype iPSC and iPSC generated from LRRK2 (G2019S) homozygous and heterozygous patients; (3) Check that all iPSC generated from wildtype and patients fibroblasts are in fact similar to human pluripotent ESC.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     biotin; induced pluripotent stem cell line cell; fibroblast; embryonic stem cell; LRRK2 [human] leucine-rich repeat kinase 2                                                                                                                                                                  This study has relevance to Parkinson's disease, but it is not clear whether the cells came from individuals with Parkinson's disease or just those who had mutations in LRRK2.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Maybe       
Triple negative breast carcinoma   GSE21974               8  Molecular Subtype Predicts Response to Neoadjuvant Chemotherapy in Breast Cancer                                                                                  In a prospective cohort study of 32 women with primary invasive breast cancer with a measurable lesion, we obtained a tumor specimen by high speed core biopsy before and after 4 cycles of NAC with epirubicine 90mg/m2 and cyclophosphamide 600mg/m2 every 3 weeks, followed by 4 cycles of docetaxel 100mg/m2. Total RNA was extracted from tumor specimens and the whole transcriptome was quantified with Agilent’s 44K single color microarray interrogating 41000 unique human genes. Tumor lesions were ultrasonographically measured to assess response using Sinn criteria. Genes significant different and significant changing during chemotherapy accompanying and predicting response were searched.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                Whole genome expression of breast cancer tissues were measured before and after four cycles neoadjuvant chemotherapy                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  female; breast; breast cancer; 4'-epidoxorubicin; NAC; docetaxel trihydrate; cyclophosphamide                                                                                                                                                                                                 From the paper (https://pubmed.ncbi.nlm.nih.gov/21769435) abstract: "Ten tumors were classified as basal-like and 22 tumors were
classified as non-basal-like." According to the Mondo Ontology entry for basal-like breast carcinoma, it is a "biologic subset of breast carcinoma defined by high expression of genes characteristic of basal epithelial cells, including KRT5 and KRT17, annexin 8, CX3CL1, and TRIM29, and usually by lack of expression of the estrogen receptor (ER), progesterone receptor (PR), and human epidermal growth factor receptor 2 (HER2)." There is not clear overlap, but it is not surprising that this study was scored highly.   No          
Triple negative breast carcinoma   GSE22513               8  Markers of Taxane Sensitivity in Breast Cancer                                                                                                                    The purpose of this study was to identify molecular markers of pathologic response to neoadjuvant paclitaxel/radiation treatment, protein and gene expression profiling were done on pretreatment biopsies. Patients with high-risk, operable breast cancer were treated with three cycles of paclitaxel followed by concurrent paclitaxel/radiation. Tumor tissue from pretreatment biopsies was obtained from 19 of the 38 patients enrolled in the study. Protein and gene expression profiling were done on serial sections of the biopsies from patients that achieved a pathologic complete response (pCR) and compared to those with residual disease, non-pCR (NR). Proteomic and validation immunohistochemical analyses revealed that α-defensins (DEFA) were overexpressed in tumors from patients with a pCR. Gene expression analysis revealed that MAP2, a microtubule-associated protein, had significantly higher levels of expression in patients achieving a pCR. Elevation of MAP2 in breast cancer cell lines led to increased paclitaxel sensitivity. Furthermore, expression of genes that are associated with the basal-like, triple-negative phenotype were enriched in tumors from patients with a pCR. Analysis of a larger panel of tumors from patients receiving presurgical taxane-based treatment showed that DEFA and MAP2 expression as well as histologic features of inflammation were all statistically associated with response to therapy at the time of surgery. We show the utility of molecular profiling of pretreatment biopsies to discover markers of response. Our results suggest the potential use of immune signaling molecules such as DEFA as well as MAP2, a microtubule-associated protein, as tumor markers that associate with response to neoadjuvant taxane–based therapy.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Tumor tissues from pretreatment needle biopsies from patients enrolled on a paclitaxel/radiation clinical trial were laser capture microdissected for RNA extraction and hybridization to Affymetrix microarrays. We analyzed one array (sample A) from duplicate samples from each patient. 14 subject, 2 replicates each.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           biotin; breast cancer; paclitaxel; sensitive toward                                                                                                                                                                                                                                           As shown in Table 1 of the paper (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2892225), some of the patients had triple-negative breast carcinoma.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    Yes         
Parkinson's disease                GSE36321               7  Gene expression analysis of H9 hESC derived neuron stem cells (NSC) harboring pathogenic LRRK2 (G2019S) mutation                                                  Genetic mutations on leucine-rich repeat kinase 2 (LRRK2) have been associated with an increased risk of Parkinson's disease. The Gly2019Ser (G2019S) mutation on LRRK2 gene is a relatively common cause of familial Parkinson's disease in Caucasian population.
In this study, we generated H9 hESC harboring LRRK2 (G2019S) mutation by gene knockin. Wildtype and LRRK2 mutant hESC were differentiated into NSC using a chemically defined protocol.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         LRRK2 mutant NSC were treated with or without the LRRK2 kinase specific inhibitor (LRRK2-IN-1). Global gene expression analysis was performed to assess the overall similarity of gene expression profiles among three NSC groups (wildtype; LRRK2 mutant; LRRK2 mutant with inhibitor treatment).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    biotin; derived from H9 embryonic stem cell; neuronal stem cell; LRRK2 [human] leucine-rich repeat kinase 2; Overexpression                                                                                                                                                                   This study has relevance to Parkinson's disease, but it is not clear whether the cells came from individuals with Parkinson's disease or just those who had mutations in LRRK2.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Maybe       
Parkinson's disease                GSE52912               7  Differential expression of human parthenogenic stem cells, neural stem cells and DA progenitors.                                                                  Neural stem cells (NSC) derived from human parthenogenic stem cells (hpSC) have been observed to show stronger positive functional effects than hpSC-derived dopaminergic neuron precursors (DAP) in treatment of induced Parkinson Disease in animal models. RNAseq of the two types of cells were normalized and analyzed to compare gene expression profiles.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  cDNA library of hpsC, NSC and DAP triplicates were sequenced using Illumina HiSeq 2000. The sequence reads were mapped to hg19 reference genome and hits that passed quality filters were analyzed for differential expression.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       neuronal stem cell; dopaminergic neuron; stem cell                                                                                                                                                                                                                                            This study has relevance to Parkinson's disease, but it is not clear whether the cells came from individuals with Parkinson's disease.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 Maybe       
Bipolar disorder                   GSE17612               6  Comparison of post-mortem tissue from brain BA10 region between schizophrenic and control patients.                                                               Analysis of gene expression in two large schizophrenia cohorts identifies multiple changes associated with nerve terminal function.
Schizophrenia is a severe psychiatric disorder with a world-wide prevalence of 1%. The pathophysiology of the illness is not understood, but is thought to have a strong genetic component with some environmental influences on aetiology. To gain further insight into disease mechanism, we used microarray technology to determine the expression of over 30 000 mRNA transcripts in post-mortem tissue from a brain region associated with the pathophysiology of the disease (Brodmann area 10: anterior prefrontal cortex) in 28 schizophrenic and 23 control patients.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 Post-mortem derived BA10 tissue from 28 schizophrenic and 23 control patients were compared. Age, gender, post-mortem delay and pH of brain lysates data were also captured.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          biotin; Brodmann (1909) area 10; schizophrenia; female; male                                                                                                                                                                                                                                  ChatGPT: "Schizophrenia and bipolar disorder are both complex psychiatric conditions that impact cognitive function, mood, and perception. Despite being distinct diagnoses, they share several features and have overlapping aspects in terms of symptoms, genetics, and potentially, their treatment." The models likely identified similarities in the ways these conditions are portrayed in dataset descriptions.                                                                                                                                                                                                                                                 No          
Bipolar disorder                   GSE54568               6  Expression data from human brain dorsolateral prefrontal cortex - including control samples and samples with major depression disorders (30 samples BA9_F)        Major depressive disorder is a heterogeneous illness with a mostly uncharacterized pathology. Large scale gene expression (transcriptome) analysis and genome-wide association studies (GWAS) for single nucleotide polymorphisms have generated a considerable amount of gene- and disease-related information, but heterogeneity and various sources of noise have limited the discovery of disease mechanisms. As systematic dataset integration is becoming essential, we developed methods and performed meta-clustering of gene coexpression links in 11 transcriptome studies from postmortem brains of human subjects with major depressive disorder (MDD) and non-psychiatric control subjects. We next sought enrichment in the top 50 meta-analyzed coexpression modules for genes otherwise identified by GWAS for various sets of disorders. One coexpression module of 88 genes was consistently and significantly associated with GWAS for MDD, other neuropsychiatric disorders and brain functions, and for medical illnesses with elevated clinical risk of depression, but not for other diseases (See publication for details).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               30 total samples in 15 pairs were analyzed in postmortem tissue from the dorsolateral prefrontal cortex.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              biotin; dorsolateral prefrontal cortex; Depression; major depressive disorder                                                                                                                                                                                                                 ChatGPT: "Bipolar Disorder (BD) and Major Depressive Disorder (MDD) are both mood disorders that share some common features, particularly involving depressive episodes, but they are distinct conditions with different diagnostic criteria." The models likely identified similarities in the ways these conditions are portrayed in dataset descriptions.                                                                                                                                                                                                                                                                                                           No          
Down syndrome                      GSE178687              6  RNAseq analysis of DPSC-derived neurons from Neurotypical control subjects and PWS subjects                                                                       A major challenge to the study and treatment of neurogenetic syndromes is the difficulty in gaining access to live neurons from individuals with these disorders. Although other sources of stem cells are currently available for differentiation into neurons, these can involve invasive procedures and be difficult or expensive to generate limiting their use on a broad scale, especially for rare syndromes which may not be well represented in the local population. Dental pulp stem cells (DPSC) are neural crest derived multipotent stem cells that reside deep the pulp of shed (baby) teeth and have the potential for broad use in the study of neurogenetic disease. Here, we use DPSC-derived neurons to investigate the transcriptional differences between neurotypical controls and Prader-Willi syndrome (PWS) conferred by uniparental disomy (UPD) versus deletion. Additionally, we investigated the increased autism incidence within the UPD genotype by sequencing PW-UPD patients with and without autism. Using this data, we defined a PWS molecular signature and discovered a global reduction in mitochondrial transcripts in the PW-UPD +ASD neurons.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         RNA-seq Analysis of DPSC derived neurons; 4 individuals per group of the following: neurotypical control, PWS deletion, PWS UPD -ASD, and PWS UPS +ASD                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                neuron; Prader-Willi syndrome  Prader-Willi syndrome due to paternal 15q11q13 deletion; Prader-Willi syndrome due to maternal uniparental disomy of chromosome 15; autism spectrum disorder                                                                                                   This is not a Down syndrome (DS) dataset. The word "syndrome" may have caught the model's attention. DS is conferred by trisomy, whereas PWS is conferred by disomy, so the textual similarity might have caught the model's attention. Autism spectrum disorder is much more common in people with DS than the general population, so there may have been mention of this in DS dataset descriptions.                                                                                                                                                                                                                                                                 No          
Down syndrome                      GSE211103              6  Tissue and cell-type specific molecular and functional signatures of 16p11.2 reciprocal genomic disorder across mouse brain and human neuronal models.            Chromosome 16p11.2 reciprocal genomic disorder due to recurrent copy number variants (CNVs) involves intellectual disability, autism spectrum disorder (ASD), and schizophrenia but the responsible mechanisms are not known. To systemically dissect molecular effects, we performed transcriptome profiling of 350 libraries from six tissues (cortex, cerebellum, striatum, liver, brown fat, white fat) in mouse models harboring CNVs of the syntenic 7qF3 region, as well as cellular, transcriptional, and single-cell analyses in 54 isogenic neural stem cell, induced neuron, and cerebral organoid models of CRISPR-engineered 16p11.2 CNVs. Transcriptome-wide differentially expressed genes were largely tissue, cell-type, and dosage specific, though more effects were shared between deletion and duplication and across tissue than expected by chance. The broadest effects were observed in the cerebellum (2163 differentially expressed genes) and the greatest enrichments were associated with synaptic pathways in mouse cerebellum and human induced neurons. Pathway and co-expression analyses identified energy and RNA metabolism as shared processes and enrichment for ASD-associated/loss-of-function constraint/FMRP gene sets. Intriguingly, reciprocal 16p11.2 dosage changes resulted in consistent decrements in neurite and electrophysiological features, while single-cell profiling of organoids showed reciprocal alterations to the proportions of excitatory and inhibitory GABAergic neurons. Both neuronal ratios and gene expression changes in our organoid analyses point most directly to calretinin GABAergic inhibitory neurons and the excitatory/inhibitory balance as targets of disruption in 16p11.2 carriers that may contribute to changes in neurodevelopmental and cognitive function. Collectively, our data indicate the genomic disorder involves disruption of multiple contributing biological processes, with relative impacts that are context-specific.                                                                                                                                                                                                                                                                                                                                                    bulk RNAseq based transcriptomic analyses of six tissues from 16p11.2 deletion and duplication as well as control mouse models; single cell RNAseq based transcriptomic analyses of human iPSC-derived 16p11.2 deletion and duplication as well as control organoids.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 male; intellectual disability; schizophrenia; autism spectrum disorder; neuron; GM08330 cell; KCTD13 [human] potassium channel tetramerization domain containing 13; proximal 16p11.2 microdeletion syndrome; chromosome 16p11.2 duplication syndrome; iPSC derived cell line; Heterozygous   This is not a Down syndrome (DS) dataset. Intellectual disability is a typical characteristic of people with DS and is surely mentioned in descriptions of DS datasets. Autism spectrum disorder is much more common in people with DS than the general population, so there may have been mention of this in DS dataset descriptions.                                                                                                                                                                                                                                                                                                                                 No          
